lib/config/specification.yml

```
@@ -265,6 +265,40 @@ billable_utilizations:
       foreign_key: account_membership_id
       collection: account_memberships
 
+billing_milestones:
+  attributes:
+    - id
+    - title
+    - date_to_invoice
+    - amount_in_cents
+    - workspace_id
+    - billing_type
+    - created_at
+    - updated_at
+    - currency
+    - external_reference_ids
+    - invoice_id
+    - invoice_ids
+    - story_id
+  create_attributes:
+    - title
+    - date_to_invoice
+    - amount_in_cents
+    - workspace_id
+    - story_id
+    - external_reference
+  update_attributes:
+    - title
+    - date_to_invoice
+    - amount_in_cents
+  associations:
+    workspace:
+      foreign_key: workspace_id
+      collection: workspaces
+    invoice:
+      foreign_key: invoice_id
```

Because this is a has_many relationship the foreign key should be plural.
```suggestion
      foreign_key: invoice_ids
```